src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -9095,500 +9108,134 @@ public static string M2(object o)
-  IL_004d:  ldloc.s    V_7
-  IL_004f:  brfalse.s  IL_005a
-  IL_0051:  br.s       IL_0053
-  IL_0053:  ldloc.s    V_7
-  IL_0055:  ldc.i4.1
-  IL_0056:  beq.s      IL_005c
-  IL_0058:  br.s       IL_005e
-  IL_005a:  br.s       IL_0078
-  IL_005c:  br.s       IL_0081
-  IL_005e:  br.s       IL_008a
-  IL_0060:  ldloc.1
-  IL_0061:  stloc.3
-  IL_0062:  br.s       IL_0064
-  IL_0064:  ldstr      ""Generic<long>.Color.""
-  IL_0069:  ldloc.3
-  IL_006a:  box        ""Generic<long>.Color""
-  IL_006f:  call       ""string string.Concat(object, object)""
-  IL_0074:  stloc.s    V_8
-  IL_0076:  br.s       IL_0093
-  IL_0078:  ldstr      ""Generic<object>.Color.Red""
-  IL_007d:  stloc.s    V_8
-  IL_007f:  br.s       IL_0093
-  IL_0081:  ldstr      ""Generic<dynamic>.Color.Blue""
-  IL_0086:  stloc.s    V_8
-  IL_0088:  br.s       IL_0093
-  IL_008a:  ldstr      ""None""
-  IL_008f:  stloc.s    V_8
-  IL_0091:  br.s       IL_0093
-  IL_0093:  ldloc.s    V_8
-  IL_0095:  ret
+  IL_000d:  stloc.s    V_5
+  IL_000f:  ldloc.s    V_5
+  IL_0011:  isinst     ""Generic<long>.Color""
+  IL_0016:  ldnull
+  IL_0017:  cgt.un
+  IL_0019:  dup
+  IL_001a:  brtrue.s   IL_001f
+  IL_001c:  ldc.i4.0
+  IL_001d:  br.s       IL_0026
+  IL_001f:  ldloc.s    V_5
+  IL_0021:  unbox.any  ""Generic<long>.Color""
+  IL_0026:  stloc.1
+  IL_0027:  brfalse.s  IL_002b
+  IL_0029:  br.s       IL_005e
+  IL_002b:  ldloc.0
+  IL_002c:  stloc.s    V_5
+  IL_002e:  ldloc.s    V_5
+  IL_0030:  isinst     ""Generic<object>.Color""
+  IL_0035:  ldnull
+  IL_0036:  cgt.un
+  IL_0038:  dup
+  IL_0039:  brtrue.s   IL_003e
+  IL_003b:  ldc.i4.0
+  IL_003c:  br.s       IL_0045
+  IL_003e:  ldloc.s    V_5
+  IL_0040:  unbox.any  ""Generic<object>.Color""
+  IL_0045:  stloc.2
+  IL_0046:  brfalse.s  IL_005c
+  IL_0048:  ldloc.2
+  IL_0049:  stloc.s    V_6
+  IL_004b:  ldloc.s    V_6
+  IL_004d:  brfalse.s  IL_0058
+  IL_004f:  br.s       IL_0051
+  IL_0051:  ldloc.s    V_6
+  IL_0053:  ldc.i4.1
+  IL_0054:  beq.s      IL_005a
+  IL_0056:  br.s       IL_005c
+  IL_0058:  br.s       IL_0076
+  IL_005a:  br.s       IL_007f
+  IL_005c:  br.s       IL_0088
+  IL_005e:  ldloc.1
+  IL_005f:  stloc.3
+  IL_0060:  br.s       IL_0062
+  IL_0062:  ldstr      ""Generic<long>.Color.""
+  IL_0067:  ldloc.3
+  IL_0068:  box        ""Generic<long>.Color""
+  IL_006d:  call       ""string string.Concat(object, object)""
+  IL_0072:  stloc.s    V_7
+  IL_0074:  br.s       IL_0091
+  IL_0076:  ldstr      ""Generic<object>.Color.Red""
+  IL_007b:  stloc.s    V_7
+  IL_007d:  br.s       IL_0091
+  IL_007f:  ldstr      ""Generic<dynamic>.Color.Blue""
+  IL_0084:  stloc.s    V_7
+  IL_0086:  br.s       IL_0091
+  IL_0088:  ldstr      ""None""
+  IL_008d:  stloc.s    V_7
+  IL_008f:  br.s       IL_0091
+  IL_0091:  ldloc.s    V_7
+  IL_0093:  ret
 }"
             );
         }
 
-        [Fact, WorkItem(19280, "https://github.com/dotnet/roslyn/issues/19280")]
-        public void ShareLikeKindedTemps_01()
```

Comment:
>ShareLikeKindedTemps_01 [](start = 20, length = 23)

I am not sure what happened with this test, it looks like it was deleted. Intentional?